src/Cooker/providers/GitProvider/getHashListFromHash.test.js

```diff
@@ -24,11 +24,17 @@ it('should return empty list for master', done => {
   })
 })
 
+/*
+
+  Do not think this is correct? If missing release hash we should return
+  all history as long as it is under 50 commits?
+
 it('should return an empty list for bad hash', done => {
```

Comment:
I am keeping my point here that "not a valid hash" should throw an error and abort. So this test should ensure it throws an error.